src/client/config/env.js

```diff
@@ -1,2 +1,5 @@
-// eslint-disable-next-line import/prefer-default-export
-export const { REACT_APP_GOOGLE_ANALYTICS_ID } = process.env;
+// export const {
```

Comment:
Add an issue to solve adding env files into the front end. Probably requires some Webpack stuff. @hasanagh, any ideas?